kaolin/ops/spc/spc.py

```diff
@@ -0,0 +1,399 @@
+#
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# NVIDIA CORPORATION and its licensors retain all intellectual property
+# and proprietary rights in and to this software, related documentation
+# and any modifications thereto.  Any use, reproduction, disclosure or
+# distribution of this software and related documentation without an express
+# lice
+# nse agreement from NVIDIA CORPORATION is strictly prohibited.
+#
+
+import math
+from torch import nn
+from torch.autograd import Function
+import torch
+import numpy as np
+
+from kaolin import _C
+
+
+
+def ScanOctrees(octrees, olens):
```

Comment:
Renamed to scan_octrees. However, I left in the name ScanOctrees in bindings.cpp. I did the same for all public facing API calls in spc.py.

Reply:
I think the string part as the python name declaration should follow python style convention. I'm ok with Cpp name since we don't have a strict style, for ScanOctrees something like:
```
ops_spc.def("scan_octrees", &ScanOctrees);
```